insonmnia/miner/gpu/nvidia_tuner.go

```diff
@@ -19,9 +20,23 @@ type nvidiaTuner struct {
 }
 
 func (g *nvidiaTuner) Tune(hostconfig *container.HostConfig) error {
-	// NOTE: driver name depends on UNIX socket name which Docker uses to connect to a driver
-	hostconfig.VolumeDriver = g.options.volumeDriverName
-	hostconfig.Binds = append(hostconfig.Binds, g.options.volumeName()+":/usr/local/lib/nvidia:ro")
+	hostconfig.Mounts = append(hostconfig.Mounts, mount.Mount{
+		Type:         mount.TypeVolume,
+		Source:       g.options.volumeName(),
+		Target:       "/usr/local/lib/nvidia",
```

Comment:
Seems constant and possibly configurable. I'd rather reuse this code with the AMD one.